lib/crypto/crypto_compat.h

```diff
@@ -11,7 +11,9 @@ int crypto_compat_RSA_valid_size(const RSA * const key);
 
 /**
  * crypto_compat_RSA_import(key, n, e, d, p, q, dmp1, dmq1, iqmp):
- * Import the given BIGNUMs into the RSA ${key}.
+ * Import the given BIGNUMs into the RSA ${key}.  If this function fails,
```

Comment:
This is the only change that I'm not completely certain about.

I mean, ultimately I think it's a good change, and I can't think of a better way to organize it.  But the API comment is a bit awkward, which makes me think that it *isn't* the best way to handle this.

Reply:
I think it's fine apart from the grammar.  "If this function fails, free any BIGNUMs which have not been imported into ${key} (but do not free ${key} itself)."  Imperative case or whatever it's called.